Makefile

```diff
@@ -267,7 +267,14 @@ test-restart-service:
 	@go test -timeout=45m -v -count=1 ./entrypoint/restartservice/...
 
 #========================= TestCode Static Quality Check =========================#
-.PHONY: vet-lint
-vet-lint:
-	@echo "Running go vet and lint"
-	@go vet ./${TEST_DIR} && golangci-lint run --tests
\ No newline at end of file
+#.PHONY: vet-lint
+#vet-lint:
+#	@echo "Running go vet and lint"
+#	@go vet ./${TEST_DIR} && golangci-lint run --tests
+
+.PHONY: pre-commit
+pre-commit:
+	@gofmt -s -w .
+	@goimports -w .
+	@go vet ./...
+	@golangci-lint run --tests ./...
```

Comment:
I'm also thinking maybe we just ensure the golangci-lint runs and blocks if this is working well -- that way we don't have to all have golangci-lint installed locally. But I'm fine either way

Reply:
yes, for this u will have to have this and also imports installed 
but the idea of this pre commit is kind of that , so we all start to have the same local formatting , avoiding adding unnecessary warnings on lint CI also facilitating lives for reviewers that dont need to look for more basic stuff on code review